test/test_helper.rb

```diff
@@ -3,6 +3,11 @@
 require 'active_support/core_ext/kernel/reporting'
 require 'active_model'
 require 'rails'
+begin
+  require 'mongoid'
+rescue LoadError => e
+  puts e
```

Comment:
I think there's no need to output the exception, so we can change it to:

```ruby
begin
  require 'mongoid'
rescue LoadError
end
```

Can you please apply this change? Thanks!